taskcluster/ci/toolchain/kind.yml

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - fetch
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+
+job-defaults:
+    run:
+        using: index-search
+    run-on-projects: []
+    worker-type: always-optimized
```

Comment:
We should match what we did elsewhere here.

Reply:
Done.